icons/cat.svg

```diff
@@ -0,0 +1 @@
+<svg width="24" height="24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M14 5.256A8.148 8.148 0 0 0 12 5a9.04 9.04 0 0 0-2 .227M20.098 10c.572 1.068.902 2.24.902 3.444C21 17.89 16.97 21 12 21s-9-3-9-7.556c0-1.251.288-2.41.792-3.444m-.042 0S2.11 3.58 3.5 3C4.89 2.42 8 3 9.781 5m10.391 5.002s1.64-6.42.25-7c-1.39-.58-4.5 0-6.282 2M8 14v.5m8-.5v.5" stroke="#000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/><path d="M11.25 16.25h1.5L12 17l-.75-.75Z" stroke="#000" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
The design itself is perfect as it is, I think, but the formatting is not quite right, be sure to follow the same [global attributes](https://lucide.dev/docs/icon-design-guide#global-attributes) and the same format as other icons, you can also optimize it using [SVGOMG](https://jakearchibald.github.io/svgomg/). For example:
```suggestion
<svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M14 5.256A8.148 8.148 0 0 0 12 5a9.04 9.04 0 0 0-2 .227M20.098 10c.572 1.068.902 2.24.902 3.444C21 17.89 16.97 21 12 21s-9-3-9-7.556c0-1.251.288-2.41.792-3.444m-.042 0S2.11 3.58 3.5 3C4.89 2.42 8 3 9.781 5m10.391 5.002s1.64-6.42.25-7c-1.39-.58-4.5 0-6.282 2" />
  <path d="M8 14v.5m8-.5v.5" />
  <path d="M11.25 16.25h1.5" />
  <path d="M12 17l-.75-.75Z" />
</svg>
```